test/form/samples/instanceof/_expected.js

```diff
@@ -0,0 +1,3 @@
+console.log('retained');
+
+console.log('retained');
```

Comment:
Shouldn't there also be `console.log('effect')`, because the effect should be retained?

Reply:
You are right, not sure how I missed this this morning. In any case, this is more of a draft that needs more polishing in other places as well.

Reply:
Everything should be working as expected now